word_language_model/main.py

```diff
@@ -92,7 +109,17 @@ def batchify(data, bsz):
 ###############################################################################
 
 ntokens = len(corpus.dictionary)
-model = model.RNNModel(args.model, ntokens, args.emsize, args.nhid, args.nlayers, args.dropout, args.tied).to(device)
+if args.model == 'Transformer':
+	model = model.TransformerSeq2Seq(ntokens, ntokens, nhead=args.transformer_head, 
```

Comment:
RNNModel isn't using keyword arguments to pass ntokens, dropout etc. - We could be consistent so it's easier to read.